ibm/acctest/acctest.go

```diff
@@ -203,43 +203,47 @@ var (
 // For Power Colo
 
 var (
-	Pi_auxiliary_volume_name        string
-	Pi_cloud_instance_id            string
-	Pi_dhcp_id                      string
-	Pi_host_group_id                string
-	Pi_host_id                      string
-	Pi_image                        string
-	Pi_image_bucket_access_key      string
-	Pi_image_bucket_file_name       string
-	Pi_image_bucket_name            string
-	Pi_image_bucket_region          string
-	Pi_image_bucket_secret_key      string
-	Pi_image_id                     string
-	Pi_instance_name                string
-	Pi_key_name                     string
-	Pi_network_name                 string
-	Pi_network_id                   string
-	Pi_network_interface_id         string
-	Pi_placement_group_name         string
-	Pi_replication_volume_name      string
-	Pi_resource_group_id            string
-	Pi_sap_image                    string
-	Pi_shared_processor_pool_id     string
-	Pi_snapshot_id                  string
-	Pi_spp_placement_group_id       string
+	Pi_auxiliary_volume_name          string
+	Pi_cloud_instance_id              string
+	Pi_dhcp_id                        string
+	Pi_host_group_id                  string
+	Pi_host_id                        string
+	Pi_image                          string
+	Pi_image_bucket_access_key        string
+	Pi_image_bucket_file_name         string
+	Pi_image_bucket_name              string
+	Pi_image_bucket_region            string
+	Pi_image_bucket_secret_key        string
+	Pi_image_id                       string
+	Pi_instance_name                  string
+	Pi_key_name                       string
+	Pi_network_id                     string
+	Pi_network_interface_id           string
+	Pi_network_name                   string
+	Pi_network_security_group_id      string
+	Pi_network_security_group_rule_id string
+	Pi_placement_group_name           string
+	Pi_remote_id                      string
+	Pi_remote_type                    string
+	Pi_replication_volume_name        string
+	Pi_resource_group_id              string
+	Pi_sap_image                      string
+	Pi_shared_processor_pool_id       string
+	Pi_snapshot_id                    string
+	Pi_spp_placement_group_id         string
 	Pi_storage_connection           string
```

Comment:
I think the spacing here got messed up somehow.

Reply:
updated